gradle.properties

```diff
@@ -17,6 +17,11 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 #
 
+# Versioning:
+GROUP=com.airthings.lib
+LIBRARY_VERSION=0.2.6-spm
```

Comment:
Remind me, the `-spm` suffix is a requirement by kmmBridge, correct?
What would be the name/version of non-iOS libs?

Reply:
I'm thinking that we can just remove the suffix?

Reply:
I agree; and if I remember correctly, kmmBridge added it automatically anyway.